ci-operator/step-registry/openshift-qe/perfscale-gcp-data-path-fw/openshift-qe-perfscale-gcp-data-path-fw-commands.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+
+# Configure gcloud
+GOOGLE_PROJECT_ID="$(< ${CLUSTER_PROFILE_DIR}/openshift_gcp_project)"
+export GCP_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/gce.json"
+sa_email=$(jq -r .client_email ${GCP_SHARED_CREDENTIALS_FILE})
+if ! gcloud auth list | grep -E "\*\s+${sa_email}"
+then
+  gcloud auth activate-service-account --key-file="${GCP_SHARED_CREDENTIALS_FILE}"
+  gcloud config set project "${GOOGLE_PROJECT_ID}"
+fi
+
+REGION="${LEASED_RESOURCE}"
+echo "Using region: ${REGION}"
+
+CLUSTER_NAME=$(oc get infrastructure cluster -o json | jq -r '.status.apiServerURL' | awk -F.  '{print$2}')
```

Comment:
Related discussion https://github.com/openshift/release/pull/44173#discussion_r1355334926

Reply:
just thru, we have dittybopper jobs before this so it is expected api server should be available at this stage.
```
INFO[2023-10-11T17:12:48Z] Running step perfscale-gcp-data-path-6nodes-periodic-XXXXXXXXXXXX-perfscale-gcp-data-path-fw. 
INFO[2023-10-11T17:14:10Z] Step perfscale-gcp-data-path-6nodes-periodic-XXXXXXXXXXXX-perfscale-gcp-data-path-fw succeeded after 1m22s. 
```

Reply:
Shouldn't block us but as we keep learning from the related issue, we may need to revisit how we are retrieving CLUSTER_NAME if needed.